src/components/textarea/textarea.stories.tsx

```diff
@@ -80,12 +81,71 @@ export const UnenforcedCharacterLimitStory: ComponentStory<
       onChange={({ target }) => setValue(target.value)}
       characterLimit="50"
       enforceCharacterLimit={false}
-      warnOverLimit
     />
   );
 };
 UnenforcedCharacterLimitStory.parameters = { chromatic: { disable: true } };
 
+export const TranslationsCharacterLimitStory: ComponentStory<
+  typeof Textarea
+> = () => {
+  const [value, setValue] = useState("");
+  return (
+    <>
+      <I18nProvider
```

Comment:
I think that example with english translations is kinda redundant considering the fact that they're exactly the same as the ones provided by us by default